src/test/java/gravitas/task/EventTest.java

```diff
@@ -0,0 +1,69 @@
+package gravitas.task;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Maybe you can avoid using wildcard imports but rather specify the imported class